index.src.html

```diff
@@ -5742,9 +5757,11 @@
                 3. If a |timestamp| can otherwise be trivially generated from
                     metadata without further decoding, assign that to |timestamp|.
                 4. Otherwise, assign `0` to |timestamp|.
-        4. Assign {{ImageDecodeResult/image}} with the result of running the
-            [=Create a VideoFrame=] algorithm with |output|, |timestamp|, and
-            |duration|.
+        4. If {{ImageDecoder/[[encoded data]]}} contains orientation metadata
+            describe it as |rotation| and |flip|.
```

Comment:
we probably need to specify `rotation=0` and `flip=false` for cases when `[[encoded data]]` doesn't contain orientation metadata.